src/OrchardCore.Modules/OrchardCore.UrlRewriting/AdminMenu.cs

```diff
@@ -32,7 +32,7 @@ protected override ValueTask BuildAsync(NavigationBuilder builder)
 
         builder
             .Add(S["Tools"], tools => tools
-                .Add(S["URL rewriting"], S["URL rewriting"].PrefixPosition(), rewriting => rewriting
+                .Add(S["URL rewriting"], S["URL Rewriting"].PrefixPosition(), rewriting => rewriting
```

Comment:
@hishamco you changed one of the two. 

It should have been

.Add(S["URL Rewriting"], S["URL Rewriting"].PrefixPosition(), rewriting => rewriting